lib/sycamore/sycamore/writer.py

```diff
@@ -563,6 +563,8 @@ def neo4j(
         uri: str,
         auth: Union[tuple[Any, Any], "Auth", "AuthManager", None],
         import_dir: str,
+        use_auradb: bool = False,
+        s3_session: Any = None,
```

Comment:
What is s3_session suppposed to be? Ordinarily I would think you can do this all with boto3 / boto3.S3.Client ?

Reply:
from boto.session import Session 

thats the type annotation we use now.